src/debugger/launch.ts

```diff
@@ -161,8 +168,8 @@ export function createTestConfiguration(
             program: `${folder}/.build/debug/${ctx.swiftPackage.name}PackageTests.xctest`,
             cwd: folder,
             env: {
-                Path: `${xcTestPath};${process.env.Path}`,
                 ...testEnv,
+                Path: `${xcTestPath};${testEnv.Path}`,
```

Comment:
By doing this you are assuming `testEnv` has a path variable. On macOS and Linux it doesn't.

Reply:
Isn’t this nested in `if (process.platform == "win32")`?